docs/flashbots-mev-share/wallets/integration-guide.mdx

```diff
@@ -0,0 +1,54 @@
+---
```

Comment:
Nit: Call it "wallets and dapps"? Don't want to exclude the dapps

Reply:
Also FYI the quick start guide has been updated https://www.notion.so/Quick-start-guide-Earn-kickbacks-from-MEV-share-227a46a52d04418a9fdfa71b3c04ef9e